app/views/projects/show.html.erb

```diff
@@ -11,7 +11,71 @@
   <%= @project.created_at %>
 </p>
 
-<%= render "samples/samples", samples:@samples %>
+<h1>Samples</h1>
+
+<% if @samples.count == 0 %>
+  No samples yet.
+<% else %>
+  <div class="sample-container">
+  <table>
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>ID</th>
+        <th>Uploader</th>
+        <th>Project</th>
+        <th>Sample Status</th>
+        <th>Pipeline Run status</th>
+        <th>Job Id </th>
+        <th>Job Submitted At</th>
+        <th> Job Queue </th>
+        <th> Memory Requirement </th>
+        <th>&nbsp;</th>
+      </tr>
+    </thead>
+
+    <tbody>
+      <% @samples.each do |sample| %>
+        <tr class=<%= sample_status_display_for_hidden_page(sample)%>>
+          <td><%= link_to sample.name, sample %></td>
+          <td><%= sample.id %></td>
+          <td class='small'><%= sample.user.email if sample.user %></td>
+          <td><%= link_to sample.project.name, sample.project %></td>
+          <td><%= sample.status %>
+            <% if sample.status.to_s == 'created' and sample.created_at < 30.minutes.ago %>
+                <%= button_to 'reupload', reupload_source_sample_path(sample), method: :put %>
+            <% end %>
+            <% if Rails.env == 'staging' %>
```

Comment:
this is the only new line. the rest was just moved out of app/views/samples/_samples.html.erb